docs/developer/conventions/test/online.rst

```diff
@@ -95,20 +95,124 @@ Example fixture ~/.geotools/postgis/local.properties::
 In windows you cannot create a ".geotools" folder!
 
 1. Open up a CMD window
-2. cd to your home directory and use mkdir
-    
-    C:\Documents and Settings\Fred>mkdir .geotools
-
-3. And set up any fixtures you need:
-    
-    C:\Documents and Settings\Fred>cd .geotools
-    C:\Documents and Settings\Fred\.geotools>mkdir postgis
-    C:\Documents and Settings\Fred\.geotools>cd postgis
-    C:\Documents and Settings\Fred\.geotools\postgis>notepad typical.properties
-
-4. And use the as a guide: https://github.com/geotools/geotools/tree/master/build/fixtures
+2. cd to your home directory and use mkdir::
+
+    C:\\Documents and Settings\\Fred\>mkdir .geotools
+
+
+3. And set up any fixtures you need::
+
+    C:\\Documents and Settings\\Fred>cd .geotools
+    C:\\Documents and Settings\\Fred\\.geotools>mkdir postgis
+    C:\\Documents and Settings\\Fred\\.geotools>cd postgis
+    C:\\Documents and Settings\\Fred\\.geotools\\postgis>notepad typical.properties
+
+
+4. And use these as a guide: https://github.com/geotools/geotools/tree/master/build/fixtures
    
    Examples:
    
    * PostgisOnlineTestCase - Abstract Testcase class which connects to a specified database and creates a datastore
    * PostgisPermissionOnlineTest - Simple online test which makes use of PostgisOnlineTestCase
+
+
+Setting up a database for online testing
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You can use `docker <https://www.docker.com/>`_ to run a database flavour of your choice,
+some examples are given below. Also the Travis-CI build script shows how to install / run
+SQL Server and Oracle XE (see `.travis.yml <https://github.com/geotools/geotools/blob/master/.travis.yml>`_).
+Using docker will prevent the hassle of local installation on your computer possibly messing up your configuration.
+
+Oracle XE
+_________
+
+Oracle Database Express Edition (XE) is an unsupported version
```

Comment:
These instructions may be good to have in the README.md of the oracle plugin itself.

Reply:
I disagree, the readme's are not exposed through search engines and having both will lead to one of them always being forgotten.

The readme's have little or no exposure and there is no consensus on what one should be able to find in the readme of a module - if one exists at all - eg. neither the jdbc-oracle nor jdbc-sqlserver even have a readme

in fact the only jdbc module with a readme is jdbc-hana and contains no such information.

Reply:
+1 on better search engine support